conan/tools/files/files.py

```diff
@@ -13,9 +13,11 @@
 import six
 
 from conan.tools import CONAN_TOOLCHAIN_ARGS_FILE, CONAN_TOOLCHAIN_ARGS_SECTION
+from conan.tools.apple.apple import is_apple_os
```

Comment:
This import here doesn't apply?

Reply:
yes, sorry, that slipped through, I'll fix that and take it to develop after it